kong/plugins/oidc/schema.lua

```diff
@@ -4,6 +4,7 @@ return {
     client_id = { type = "string", required = true },
     client_secret = { type = "string", required = true },
     discovery = { type = "string", required = true, default = "https://.well-known/openid-configuration" },
+    introspection_endpoint = { type = "string", required = false },
```

Comment:
IMO, the Introspection Endpoint should be fetched from the OIDC Discovery Endpoint.

Reply:
Nope -> in OAuth only AS you will not have discovery uri

Reply:
That's a plugin for OIDC, so you can assume, that there must be a Discovery Endpoint

Reply:
introspection however is an optional extension to OAuth 2.0 and OIDC Discovery in its turn is OpenID Connect specific and predates the standardization of the introspection OAuth 2.0 extension, so OIDC Discovery hasn't standardized a key for the introspection endpoint

now there's WIP here https://tools.ietf.org/html/draft-ietf-oauth-discovery that standardizes Discovery for OAuth 2.0 and that includes the introspection endpoint... thats still TBD

Reply:
My point exactly - Thank You !